src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './utils/animations';
+
 body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin #000 dotted;
+  border-radius: 50%;
+
+  &__center {
+    height: 5vmin;
+    width: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    position: relative;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+
+    transform-origin: bottom;
+    animation: stopwatch-hend-animation 60s linear infinite;
+  }
+
+  &__minute-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    bottom: 50%;
+
+    transform-origin: bottom;
+    animation: stopwatch-hend-animation 3600s steps(60) infinite;
+  }
+
+  &--speed-up {
+    .stopwatch__minute-hand {
+      animation: stopwatch-hend-animation 600s steps(60) infinite;
+    }
+
+    .stopwatch__seconds-hand {
+      animation: stopwatch-hend-animation 10s linear infinite;
```

Comment:
Check the implementation of the 'stopwatch-hend-animation' for the seconds hand when the '.stopwatch--speed-up' modifier class is applied.